apps/explorer/src/pages/AddressInfoPage/index.tsx

```diff
@@ -197,7 +198,7 @@ const AddressInfoPage = () => {
             }
           />
           <InfoGrid.Cell
-            label={t('Fiat price')}
```

Comment:
The string "Fiat price" should be removed from the translation.json file

Reply:
Yeah I asked myself about this, thinking it may be good to keep translations for a pretty generic string that may be reused somewhere else. 
But I guess it's indeed better to keep the translation files clean and in sync with what is actually used in-app.

Reply:
Done

Reply:
It's not only that, but it reduces the work of translators.